docs/content/contributing.md

```diff
@@ -42,6 +42,18 @@ system_arguments[:tag] = :h1
 system_arguments[:tag] = fetch_or_fallback(TAG_OPTIONS, tag, DEFAULT_TAG)
 ```
 
+## Running the demo app
```

Comment:
🙇  Thank you

Reply:
this should be covered by the `### Storybook / Documentation` section 🤔 
maybe we should move it up? `script/dev` should run everything you need

Reply:
change uploaded 👍